artginzburg/MiddleClick-Sonoma
# Accept Right Mouse Down/Up events as a basic tapped event

Issue: https://github.com/artginzburg/MiddleClick-Ventura/issues/13

- Normally three-figer-click is performed by putting your three fingers at the trackpad and pushing
- Sometimes when performing three-finger-click straight away, without putting three fingers on the trackpad first, three-finger-click event tap doesn‘t work and opens context menu instead because the system interprets it as a 2-fingers click
- The PR adds adds attitional kCGEventRightMouseDown/Up interception that fixes the issue

Steps to test:
- Open a new browser Tab
- Point mouse cursor on the tab
- Perform three-finger-click instantly: without putting your fingers to the trackpad first, repeat several times: the tab should close
